Remove enable_semi_sync from docs and add cross_cell durability policy (#1103)
vitessio/website: content/en/docs/15.0/reference/programs/vttablet.md

| --enable_hot_row_protection |  | If true, incoming transactions for the same row (range) will be queued and cannot consume all txpool slots. |
| --enable_hot_row_protection_dry_run |  | If true, hot row protection is not enforced but logs if transactions would have been queued. |
| --enable_replication_reporter |  | Register the health check module that monitors MySQL replication |

> | --enable_semi_sync |  | DEPRECATED - Set the correct durability policy in the keyspace information instead |

Reviewer: ```suggestion
| --enable_semi_sync |  | DEPRECATED - Set the correct durability policy on the keyspace instead. |
```

Author: Fixed
